src/inspector.rs

```diff
@@ -520,6 +528,9 @@ where
 pub trait V8InspectorClientImpl: AsV8InspectorClient {
   fn base(&self) -> &V8InspectorClientBase;
   fn base_mut(&mut self) -> &mut V8InspectorClientBase;
+  fn base_ptr(this: *const Self) -> *const V8InspectorClientBase
+  where
+    Self: Sized;
```

Comment:
Same here. Should be `unsafe` like in the other trait

Reply:
Adjusted.